src/config-loader/config-loader.ts

```diff
@@ -161,6 +162,8 @@ export const setup = (app: Express.Application) => {
 
 							const apiRoute = `/${model.apiRoot}/*`;
 							app.options(apiRoute, (_req, res) => res.status(200).end());
+							app.all(apiRoute, webResource.multerPinejs);
```

Comment:
Hi @fisehara, the goal of these changes is to be able to receive streamed content. The approach here is to add multer support for all routes; the multer support only turns payloads received as `multipart/form-data` into `req.body` properties, so that other handlers can process it. This way, `handleODataRequest` requires no changes. It will process a, let's say, `req.body.picture` property where `req.body.picture.data` will be a Buffer.

Reply:
@fisehara This handler adds multer to `apiRoute`, and a new handler populates some variables if the request includes a file.

This allows pinejs to process `multipart/form-data`.

I don't have any specific performance data of how much overload multer adds but should be pretty efficient since it's the official `express` solution.

If you think this is important to validate, I'll try to run some tests to get some data